examples/demo/client/main.go

```diff
@@ -50,47 +55,65 @@ func initProvider() func() {
 		otelAgentAddr = "0.0.0.0:4317"
 	}
 
-	exp, err := otlp.NewExporter(ctx, otlpgrpc.NewDriver(
-		otlpgrpc.WithInsecure(),
-		otlpgrpc.WithEndpoint(otelAgentAddr),
-		otlpgrpc.WithDialOption(grpc.WithBlock()), // useful for testing
-	))
-	handleErr(err, "failed to create exporter")
+	metricClient := otlpmetricgrpc.NewClient(
+		otlpmetricgrpc.WithInsecure(),
+		otlpmetricgrpc.WithEndpoint(otelAgentAddr))
```

Comment:
Can the example applications be simplified by using the `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable instead of `OTEL_AGENT_ENDPOINT`?